src/generic/generic-component-listing/generic-component-listing.scss

```diff
@@ -31,8 +31,12 @@
         width: 50%;
       }
 
-      .ecl-list-item__link {
+      .ecl-list-item:nth-child(2n + 0) .ecl-list-item__link {
```

Comment:
To be honest, it's easy to understand what's going on here without reading aside. A comment or two what are these calculations meant to represent will be helpful. There's also the `nth-of-type(odd)` and `nth-of-type(even)` syntax which makes it a bit more obvious without the calculations.